src/terraboot/core.clj

```diff
@@ -13,6 +13,13 @@
 (def current-coreos-ami "ami-1807e377")
 (def ec2-ami "ami-bc5b48d0")
 
+(def all-external "0.0.0.0/0")
```

Comment:
the ami you mean? yes, good point
(the all-external is a generic "all addresses everywhere")